### PR TITLE
Fixing perplexed.md

### PR DESCRIPTION
**Pull request by shellygr**
See commit description.

### CODE REVIEW

File: docs/confluence/perplexed.md [hunk 1]
```diff
@@ -77,15 +79,29 @@ At this point, if sanity passed and in reasonable times (not more than 1-2 minut
     
 5.  Check one rule at a time and try to increase `-depth=15`. This could help if the code contains many branches.
     
-6.  It could be that internal function summaries were not applied, so if you provided those and there is no change in run times, it could be there’s a muted error there. ([thomas](https://certora.atlassian.net/wiki/people/60919dc2c87b550069bccbd4?ref=confluence) can you say how to debug this?)
+6.  It could be that internal function summaries were not applied, so if you provided those and there is no change in run times, please report to Certora.
```
Comment (mdgeorge4153) [inline docs/confluence/perplexed.md:82]: There's a new option that lists the summarization candidates (`-showInternalMethods`); we might suggest that people check that list.  There's a PR for documenting the flag: https://github.com/Certora/Documentation/pull/34

**Review by mdgeorge4153 — APPROVED** [review]
I think this is a great change, thank you.  I suggested some improvements.  I'll also do a more thorough review while migrating to the user guide during the documentation sprint.

File: docs/confluence/perplexed.md [hunk 2]
```diff
@@ -27,11 +27,13 @@ As you can see there’s a ~2 hours difference between “Queue Time” and “F
...
+2.  If the problem is preprocessing, first try to run with `-enableEqualitySaturation=false`, and if that does not help try in addition `-simplificationDepth=10`. If both don't help, please report to Certora.
+
+3.  If you’re blocked on this contract, try to make progress by manually removing parts of the code. The code may contain libraries that add to the complication, so try to “erase” those parts of the code first and re-run.
     
-3.  Otherwise, if rules are already starting to process, check how long approximately preprocessing took. You can measure that using the “Ping” messages. A ping message is printed every minute. If preprocessing took more than 30 minutes, report to #bugs like in step 2, and either you can try removing parts of the code or just run 1 rule instead of all rules.
+4.  Otherwise, if rules are already starting to process, check how long approximately preprocessing took. You can measure that using the “Ping” messages. A ping message is printed every minute. If preprocessing took more than 30 minutes, report to Certora like in step 2, and either you can try removing parts of the code or just run 1 rule instead of all rules.
```
[the remark this answers is not in the document]
Comment (jar-ben) [reply docs/confluence/perplexed.md:34]: @mdgeorge4153 , you mean the `Duration` shown e.g. [here](https://vaas-stg.certora.com/output/1000/b36b1acefae718a2867d/?anonymousKey=01a5a828a42c8958919eab3d4e95a479c3a93c0d) (`00:03:30`)? This number and the numbers in the "table" with rules below usually does not correspond in neither directly; sometimes if you sum them you get a bigger number than Duration and in other cases vice versa. We need to revisit the time-stats reporting (there are some related JIRA tickets).

**Review by jar-ben — APPROVED** [review]
Looks good to me!

**Pull request merged**